documentation/architecture/decisions/0003-use-ecs-for-initial-beta-buildout.md

```diff
@@ -0,0 +1,37 @@
+# 3. Use Amazon ECS for initial beta build-out
+
+Date: 2018-06-21
+
+(although note that this ADR was written post hoc, a couple of months
+after the decision was made)
+
+## Status
+
+Accepted.
+
+## Context
+
+Existing self-hosted infrastructure at GDS has been managed in code
+using tools like puppet, but in a somewhat ad hoc way with each team
+doing things differently, little sharing of code, and much reinvention
+of wheels.  We would like to learn about other ways of deploying
```

Comment:
Whilst it was clear this was new territory for a lot of us on the team and we would learn from it, this paragraph reads to me that ECS was chosen mainly as a learning experiment which wasn't the major reason I gathered for using it (more as Daniel said that we thought using Docker was the recommended way to deploy Prometheus).